bot/exts/events/trivianight/trivianight.py

```diff
@@ -0,0 +1,202 @@
+import asyncio
+from json import JSONDecodeError, loads
+from random import choice
+from typing import Optional
+
+from discord import Embed
+from discord.ext import commands
+
+from bot.bot import Bot
+from bot.constants import Colours, NEGATIVE_REPLIES, POSITIVE_REPLIES
+
+from ._questions import QuestionView, Questions
+from ._scoreboard import Scoreboard, ScoreboardView
+
+
+class TriviaNight(commands.Cog):
+    """Cog for the Python Trivia Night event."""
+
+    def __init__(self, bot: Bot):
+        self.bot = bot
+        self.scoreboard = Scoreboard()
+        self.questions = Questions(self.scoreboard)
+
+    @staticmethod
+    def unicodeify(text: str) -> str:
+        """Takes `text` and adds zero-width spaces to prevent copy and pasting the question."""
+        return "".join(
+            f"{letter}\u200b" if letter not in ('\n', '\t', '`', 'p', 'y') else letter
+            for idx, letter in enumerate(text)
+        )
+
+    @commands.group(aliases=["tn"], invoke_without_command=True)
+    async def trivianight(self, ctx: commands.Context) -> None:
+        """No-op subcommand group for organizing different commands."""
+        if ctx.invoked_subcommand is None:
```

Comment:
This is what the `invoke_without_command` kwarg does.

Reply:
Ah, I didn't know that so I used both. Thanks!